### PR TITLE
EMC Compliant Standards

### PR DESCRIPTION
**Pull request by M-J-Robbins**
adding first draft of the document

### CODE REVIEW

**Comment by husseinalhammad** [comment]
Sorry @M-J-Robbins it's taking me a while to review this PR. I'll do so soon.

File: compliant-standards.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+## 2. Accessibility
+
+Accessibility is key to achieving our markup standards. When email markup is properly designed and coded, people of all abilities can use it.
+
+### 2.1 Content should fit the container
+
+Email content should respond to fit the size of the container it is loaded in. A recipient should not have to scroll on more than one axis to see parts of the email, nor should content be cut off.
+
+> Content should adjust to fit the available space with fluid content or @media queries.
```
Comment (husseinalhammad) [inline compliant-standards.md:41]: I don't think block quotes is the most suitable way to format this. The same applies to the other points. 

If it is an example, perhaps having "Examples" as a subheading would make things clearer. If it is further explanation, nothing (including the block quote) is required.

File: compliant-standards.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+
+### 1.4 What this document is not
+
+This document is not a “how to” guide on how you can be compliant with these standards.
+
+This document does not cover email design or content choices.
+
+This document does not cover email sending protocols.
+
+This document does not take into consideration current levels of markup support; these are targets to strive for.
```
Comment (husseinalhammad) [inline compliant-standards.md:31]: This looks like a list. Perhaps we should format it as a list?

File: compliant-standards.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+
+Unsupported elements should keep their supported attributes and be replaced with a semantically neutral element or [valid custom element](https://html.spec.whatwg.org/multipage/custom-elements.html#valid-custom-element-name).
+
+> Replace `<main class="foo">` with `<div class="foo">`
+
+### 3.4 Group code to fallback when not supported
+
+Code that is dependent on support of other code should be grouped in a way that it can fallback without showing broken partial support.
+
+> Issues can arise when `display: flex` is supported but not `flex-wrap`. We can use things like @supports targeting to help group these so they fallback cleanly.
```
Comment (husseinalhammad) [inline compliant-standards.md:109]: > We can use things like

Perhaps "approaches" is a better word than "things" here. Not very important to change.

File: compliant-standards.md [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+
+The Email Markup Consortium (EMC) is working to improve email markup, the code used to structure and style content within an email. This document lays out the standards of what that markup should look like.  
+  
+This document is aimed at top level ideas of what the email markup should do, not specific ways to code.
+
+### 1.2 Who is this document for?
+
+This document is for anyone creating email markup, processing email markup, or displaying email markup. This includes (but isn’t limited to) email developers, email creation tools, email sending tools, and email clients.
+
+For this to work it will require developers to make improvements to the markup at the point of creation. As long as that markup is valid it should remain in a recognisable state with minimal changes through to the point the recipient opens the email. This means any processing of that markup by an ESP, mail server, email client or any other technology that touches it, should also follow the guidelines to preserve the standard.
```
Comment (husseinalhammad) [inline compliant-standards.md:15]: Many terms are used here. Do we need to link to the [glossary of terms](https://github.com/email-markup-consortium/email-markup-consortium/blob/main/glossary.md)?

**Review by husseinalhammad — COMMENTED** [review]
Overall looks very good, @M-J-Robbins. Just a couple of formatting updates, then it's good to merge!

File: compliant-standards.md (hunk 1 above)
Comment (M-J-Robbins) [reply compliant-standards.md:41]: Yeah it's a tricky one, i was thinking of a way to seperate it visually but it's hard in markdown, I think a subheading is a good idea.  Also if I'm planning on only doing it visually it's not accessible

File: compliant-standards.md (hunk 4 above)
Comment (M-J-Robbins) [reply compliant-standards.md:15]: Yeah it's a good point I've added a few but it's hard to link directly to the terms. This should be easier when these pages are moved to the website.

**Comment by M-J-Robbins** [comment]
@husseinalhammad great feedback, I've made some updates based on your notes. Let me know what you think 👍

**Review by husseinalhammad — APPROVED** [review]

**Comment by husseinalhammad** [comment]
Looks great, @M-J-Robbins. Thank you!

**Pull request merged**